init_db/001-users_collection.js

```diff
@@ -55,6 +55,30 @@ let users = {
             is_banned: {
                 bsonType: 'bool',
                 description: 'must be a boolean'
+            },
+            rate_count: {
```

Comment:
o co chodzi z rate_count rate_value i avg_rating. Można oceniać userów?

Reply:
To średnia ocen wystawionych alkoholom.

Reply:
A po co coś takiego?

Reply:
Daro dał taki widok usera w figmie i go razem zaakceptowaliśmy. Chyba jeszcze przed wakacjami XD